samples/migrations/migrate_chromadb_vectorstore_to_alloydb.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python
+
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+from typing import Any, Iterator
+
+"""Migrate Chroma to LangChain AlloyDBVectorStore.
+Given a Chroma collection, the following code fetches the data from Chroma
+in batches and uploads to an AlloyDBVectorStore.
+"""
+
+# TODO(dev): Replace the values below
+PROJECT_ID = "YOUR_PROJECT_ID"
+REGION = "YOUR_REGION"
+CLUSTER = "YOUR_CLUSTER_ID"
+INSTANCE = "YOUR_INSTANCE_ID"
+DB_NAME = "YOUR_DATABASE_ID"
+DB_USER = "YOUR_DATABASE_USERNAME"
+DB_PWD = "YOUR_DATABASE_PASSWORD"
```

Comment:
nit: typically we add example values like "my-project-id", "us-central1", "my-cluster", "postgres", etc.